packages/grafana-toolkit/src/cli/tasks/plugin.build.ts

```diff
@@ -1,154 +1,155 @@
 
-import { useSpinner } from '../utils/useSpinner';
-import { testPlugin } from './plugin/tests';
-import { bundlePlugin as bundleFn, PluginBundleOptions } from './plugin/bundle';
+const { access, copyFile, readFile, writeFile } = fs;
+const { COPYFILE_EXCL, F_OK } = fsConstants;
+const rimraf = promisify(rimrafCallback);
+
 interface PluginBuildOptions {
   coverage: boolean;
 }
+
 interface Fixable {
   fix?: boolean;
 }
 
 export const bundlePlugin = useSpinner<PluginBundleOptions>('Compiling...', async options => await bundleFn(options));
 
 // @ts-ignore
-export const clean = useSpinner<void>('Cleaning', async () => await execa('rimraf', [`${process.cwd()}/dist`]));
-
-export const prepare = useSpinner<void>('Preparing', async () => {
-  // Make sure a local tsconfig exists.  Otherwise this will work, but have odd behavior
-  let filePath = path.resolve(process.cwd(), 'tsconfig.json');
-  if (!fs.existsSync(filePath)) {
-    const srcFile = path.resolve(__dirname, '../../config/tsconfig.plugin.local.json');
-    fs.copyFile(srcFile, filePath, err => {
-      if (err) {
-        throw err;
-      }
-      console.log(`Created: ${filePath}`);
-    });
-  }
-  // Make sure a local .prettierrc.js exists.  Otherwise this will work, but have odd behavior
-  filePath = path.resolve(process.cwd(), '.prettierrc.js');
-  if (!fs.existsSync(filePath)) {
-    const srcFile = path.resolve(__dirname, '../../config/prettier.plugin.rc.js');
-    fs.copyFile(srcFile, filePath, err => {
-      if (err) {
-        throw err;
+export const clean = useSpinner<void>('Cleaning', async () => await rimraf(`${process.cwd()}/dist`));
+
+const copyIfNonExistent = (srcPath, destPath) =>
+  copyFile(srcPath, destPath, COPYFILE_EXCL)
+    .then(() => console.log(`Created: ${destPath}`))
+    .catch(error => {
+      if (error.code !== 'EEXIST') {
+        throw error;
       }
-      console.log(`Created: ${filePath}`);
     });
-  }
-  return Promise.resolve();
+
+export const prepare = useSpinner<void>('Preparing', async () => {
+  await Promise.all([
+    // Copy only if local tsconfig does not exist.  Otherwise this will work, but have odd behavior
+    copyIfNonExistent(
+      resolvePath(process.cwd(), 'tsconfig.json'),
+      resolvePath(__dirname, '../../config/tsconfig.plugin.local.json')
+    ),
+    // Copy only if local prettierrc does not exist.  Otherwise this will work, but have odd behavior
+    copyIfNonExistent(
+      resolvePath(process.cwd(), '.prettierrc.js'),
+      resolvePath(__dirname, '../../config/prettier.plugin.rc.js')
+    ),
+  ]);
+
+  // Nothing is returned
 });
 
 // @ts-ignore
 const typecheckPlugin = useSpinner<void>('Typechecking', async () => {
   await execa('tsc', ['--noEmit']);
 });
 
-const getTypescriptSources = () => {
-  const globPattern = path.resolve(process.cwd(), 'src/**/*.+(ts|tsx)');
-  return glob.sync(globPattern);
-};
+const getTypescriptSources = () => globby(resolvePath(process.cwd(), 'src/**/*.+(ts|tsx)'));
```

Comment:
What are globby advantages over glob?

Reply:
Built-in support for promises. Also, multiple patterns per function call.